core-common/src/test/java/org/mapstruct/test/PackagePrivateMapper.java

```diff
@@ -0,0 +1,27 @@
+/**
+ *  Copyright 2012-2017 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.test;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+interface PackagePrivateMapper {
+    PackagePrivateMapper INSTANCE = Mappers.getMapper( PackagePrivateMapper.class );
```

Comment:
This is causing checkstyle to fail. You can remove this `INSTANCE` here and it should work. See the `MappersTest`. By the way you don't have to annotate the class with `@Mapper`, the `Mappers` factory does no checks for that